modules/graph/fragment/gar_fragment_builder.h

```diff
@@ -53,7 +53,9 @@ class GARFragmentBuilder
                               std::shared_ptr<vertex_map_t> vm_ptr)
       : ArrowFragmentBaseBuilder<oid_t, vid_t, vertex_map_t>(client),
         client_(client),
-        vm_ptr_(vm_ptr) {}
+        vm_ptr_(vm_ptr) {
+    Base::set_compact_edges_(false);
```

Comment:
Brief for why change here：`gar_fragment_builder` uses this constructor to construct builder and wants the `compact_edges` to be set false, but sometimes the constructor will initialize the `compact_edges` to be true, that would make builder corrupt.

Reply:
I suggest you add an assertion and throw exception like "compacting edges is not supported when loading from GraphAr" or something like that.